src/sentry/locale/en/LC_MESSAGES/django.po

```diff
@@ -3590,7 +3590,7 @@ msgid "e.g. operations, web-frontend, desktop"
 msgstr ""
 
 #: static/sentry/app/components/createTeam/createTeamForm.jsx:45
-msgid "May contain lowercase letters, numbers, dashes and underscores."
+msgid "May only contain lowercase letters, numbers, dashes and underscores."
```

Comment:
I think you'll also need to change the text used in `static/sentry/app/components/createTeam/createTeamForm.jsx`